lib/__tests__/created_message_queue_handler.test.ts

```diff
@@ -51,8 +52,8 @@ const aCorrectFiscalCode = _getO(
 );
 const aWrongFiscalCode = "FRLFRC74E04B157" as FiscalCode;
 const anEmail = _getO(t.validate("x@example.com", EmailString).toOption());
-const anEmailNotification: INotificationChannelEmail = {
-  addressSource: NotificationAddressSource.PROFILE_ADDRESS,
+const anEmailNotification: NotificationChannelEmail = {
+  addressSource: NotificationAddressSourceEnum.PROFILE_ADDRESS,
```

Comment:
better naming: is the "Enum" suffix mandatory ?

Reply:
yeah kind of, unfortunately TypeScript cannot export both a type and an `enum` with the same name (see [here](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/api/definitions/NotificationChannelStatus.ts))